infrastructure/simulation-tool/config/default.json5

```diff
@@ -1,9 +1,10 @@
 {
-    "rollupUrl": "http://127.0.0.1:3030",
+    "rollupUrl": "https://127.0.0.1:3001",
     "nodeUrl": "http://127.0.0.1:4444",
-    "transactionsPerSecond": 12,
-    "totalTransactions": 10,
-    "numberOfAccounts": 1,
+    "chainId": 33,
+    "transactionsPerSecond": 3,
+    "totalRunningTimeSeconds": 5,
+    "numberOfAccounts": 3,
```

Comment:
is there any correlation with the numberOfAccounts, totalTransactions and transactionsPerSecond. I tried to change the configuration by using:

```
 "transactionsPerSecond": 10,
 "totalRunningTimeSeconds": 10,
```

But I always get the error `transaction nonce too high` from the rskj node.

Reply:
I think here we've arrived at the issue of executing all transactions in “parallel”. I suspect that if we execute too many txs per second the execution happens disorderly, therefore the nonce error. The highest TPS that works on my machine seems to be 6. However, when I use 6tps for 100 seconds, that too seems to be an issue. 500 seconds of 2TPS, is fine. I'd say that all this correlates with the hypothesis.

Reply:
OK. There are several issues in play, actually. This particular error, however, is caused by the [maximum number of transactions from one address per block](https://github.com/rsksmart/rskj/blob/HOP-4.4.0/rskj-core/src/main/java/co/rsk/net/handler/txvalidator/TxValidatorNonceRangeValidator.java#L52). This is set on HOP to be [16](https://github.com/rsksmart/rskj/blob/47bb17e12167db5ed81f683fe7cce871f30775a1/rskj-core/src/main/resources/reference.conf#L235), which coincides with the number of TXs at which we get this error.

Reply:
We had a discussion about this, Ill resolve the comment since we already took note about it.